contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -578,7 +578,7 @@ library ValidationLogic {
         }
 
         require(
-            params.liquidationAmount >= params.actualLiquidationAmount,
+            params.maxLiquidationAmount >= params.actualLiquidationAmount,
             Errors.LIQUIDATION_AMOUNT_NOT_ENOUGH
         );
 
```

Comment:
https://github.com/para-space/paraspace-core/blob/379d9e923a578d0b8c17778fcbdbfe8141593b26/contracts/protocol/libraries/logic/ValidationLogic.sol#L597-L600

maybe better to add another error code for this check